miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/SampleRestController.java

```diff
@@ -324,17 +297,15 @@ public SampleDto updateSample(@PathVariable("id") Long id, @RequestBody SampleDt
     return identitiesBySearchTerm;
   }
 
-  private Set<SampleDto> getSamplesForIdentityString(String identityIdentifier, String project, boolean permitPartialMatch)
+  private Set<SampleDto> getSamplesForIdentityString(String identityIdentifier, String project, boolean exactMatch)
       throws IOException {
     Collection<SampleIdentity> matches = new HashSet<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "matches". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)